querydsl-codegen/src/main/java/com/querydsl/codegen/EntitySerializer.java

```diff
@@ -620,6 +663,10 @@ public void serialize(EntityType model, SerializerConfig config,
 
     protected void serialize(EntityType model, Property field, Type type, CodeWriter writer,
             String factoryMethod, String... args) throws IOException {
+
+        String constantFieldName = constantFieldName(field.getEscapedName());
+        writer.publicStaticFinal(new ClassType(String.class), constantFieldName, "\"" + field.getName() + "\"");
```

Comment:
use Types.String instead of new ClassType(String.class)
also could these additions be toggable with an option?